packages/model-viewer/src/features/scene-graph/texture-info.ts

```diff
@@ -95,6 +95,27 @@ export class TextureInfo implements TextureInfoInterface {
     return this[$texture];
   }
 
+  get rotation(): number|null {
+    if (!this[$texture]) {
+      return null;
```

Comment:
I know glTF puts the extension on `TextureInfo`, but I'm tempted to move this API down to `Sampler` next to `wrap`, since that's what it's closely related to. Also, I have the feeling bad things will happen if a texture is referenced by more than one Info. I don't love diverging from the glTF spec, but I think it'll work better.